components/table/test/table.visual-diff.js

```diff
@@ -87,15 +94,23 @@ describe('d2l-table', () => {
 
 					describe('sticky', () => {
 
+						before(async() => {
+							await page.$eval('d2l-test-table-visual-diff', elem => elem.setAttribute('hidden', 'hidden'));
+							await page.$eval('d2l-test-table-sticky-visual-diff', elem => elem.removeAttribute('hidden'));
+						});
+
 						[
 							'one-row-thead',
-							'one-row-no-thead',
+							'one-row-no-thead-class',
+							'one-row-legacy-attr',
```

Comment:
I think this needs to be `one-row-no-thead-attr`?  And `multi-row-legacy-attr` should be `multi-row-no-thead-attr`.